CHANGES.md

```diff
@@ -3,7 +3,11 @@ Change Log
 
 ### 1.11 - 2015-07-01
 
+* Breaking changes
+  * Removed `Camera.clone` that was deprecated in 1.10.
 * Deprecated
+  * `CatmullRomSpline` has been deprecated and will be removed in Cesium 1.12.
```

Comment:
Might want to keep both and add others.  Depends on what direction is taken for tools in Viewer.  Also, GML uses b-spline and cubic spline which I believe are for roads.